data_collection/gazette/spiders/pa_barcarena.py

```diff
@@ -0,0 +1,49 @@
+import json
+from datetime import date, datetime
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PaBarcarenaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "1501303"
+    name = "pa_barcarena"
+    allowed_domains = ["domapi.tecnologia.ws"]
+    GAZETTE_NAME_CSS = "sc-6edf83ab-0 eZICqJ"
+    start_urls = ["http://domapi.tecnologia.ws/diario/consulta"]
+
+    start_date = date(2021, 11, 8)
+
+    def start_requests(self):
+
+        query = f'?data_ini={self.start_date.strftime("%Y/%m/%d")}&data_fim={self.end_date.strftime("%Y/%m/%d")}&edicao=&palavra_chave='
+
+        header = {
+            "User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:48.0) Gecko/20100101 Firefox/48.0"
+        }
+        for url in self.start_urls:
+            yield scrapy.Request(f"{url}{query}", headers=header)
+
+    def parse(self, response):
+
+        print("\n\n")
+        json_response = json.loads(response.text)
```

Comment:
A partir do momento que Response possui uma resposta textual, já é obtida uma instância de `TextResponse`, que é uma subclasse de `Response`. Desde o scrapy 2.2 (note que [usamos a versão 2.6.3](https://github.com/okfn-brasil/querido-diario/blob/03049d18839f7dfe308e651f5d98753ebb0a2a1e/data_collection/requirements-dev.txt#L234)), está disponível [o método `json()` para `TextResponse`](https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.TextResponse.json), então podemos simplificar um pouco esse trecho

```suggestion
        json_response = response.json()
```

e com isso remover a linha `import json` que passaria a ser desnecessária, já que o scrapy já cuidaria disso internamente.

Reply:
Entendi, não tenho muito conhecimento dos métodos do response, obrigado por me mostrar!